Compile error: make debug

**Describe the bug**

`make debug` fails with compile errors.

**To Reproduce**

```
$ make debug
rm -f lotus
go build -ldflags=-X="github.com/filecoin-project/lotus/build".CurrentCommit="+git6afeb2e7.dirty" -tags=debug -o lotus ./cmd/lotus
# github.com/filecoin-project/lotus/cmd/lotus
cmd/lotus/debug_advance.go:54:6: cannot use head (type *"github.com/filecoin-project/lotus/chain/types".TipSet) as type "github.com/filecoin-project/lotus/chain/types".TipSetKey in argument to api.StateCall
cmd/lotus/debug_advance.go:107:36: cannot use head (type *"github.com/filecoin-project/lotus/chain/types".TipSet) as type "github.com/filecoin-project/lotus/chain/types".TipSetKey in argument to api.MinerCreateBlock
Makefile:55: recipe for target 'lotus' failed
make: *** [lotus] Error 2
```
**Version (run `lotus --version`):**

```
$ ./lotus version
0.2.7+git6afeb2e7+api0.1.6
lotus version 0.2.7+git6afeb2e7
```

--- a/cmd/lotus/debug_advance.go
+++ b/cmd/lotus/debug_advance.go
@@ -52,7 +52,7 @@ func init() {
 					From:   addr,
 					To:     addr,
 					Method: actors.MAMethods.GetWorkerAddr,
-				}, head)
+				}, head.Key())
 				if err != nil {
 					return xerrors.Errorf("failed to get miner worker addr: %w", err)
 				}
@@ -104,7 +104,7 @@ func init() {
 
 			uts := head.MinTimestamp() + uint64(build.BlockDelay)
 			nheight := head.Height() + 1
-			blk, err := api.MinerCreateBlock(ctx, addr, head, ticket, epostp, msgs, nheight, uts)
+			blk, err := api.MinerCreateBlock(ctx, addr, head.Key(), ticket, epostp, msgs, nheight, uts)
 			if err != nil {
 				return xerrors.Errorf("creating block: %w", err)
 			}

